app/views/trails/show.html.erb

```diff
@@ -49,21 +49,41 @@
             <i class="fa-solid fa-chevron-left"></i>
           </div>
         <% end %>
+
+        <%# Bookmark icon %>
         <span class="text-decoration-none text-dark">
           <div class="d-flex justify-content-center align-items-center mx-2">
-            <%= simple_form_for [@trail, @bookmark],  html: { method: :post } do |f| %>
-              <%= f.button :button do %>
-                <i id="bookmark" class="fa-regular fa-bookmark"></i>
+            <%# <%= simple_form_for [@trail, @bookmark],  html: { method: :post } do |f| %>
+              <%#= f.button :button do %>
+                <%# <i id="bookmark" class="fa-regular fa-bookmark"></i> %>
+              <%# end %>
+            <%# button icon for bookmark <i id="bookmark" class="fa-regular fa-bookmark"></i> %>
+            <%# end %>
+            <%# <i class="fa-regular fa-bookmark"></i> %>
+            <%= link_to toggle_favorite_trail_path(@trail), remote: true, data:{turbo_method: :post} do %>
+              <% if user_signed_in? %>
+                <% if current_user.favorited?(@trail) %>
+                  <i class="fa-regular fa-bookmark fw-bold bookmark-show"></i>
+                <% else %>
+                  <i class="fa-regular fa-bookmark bookmark-show"></i>
+                <% end %>
+              <% else %>
+                <i class="fa-regular fa-bookmark bookmark-show"></i>
               <% end %>
             <% end %>
           </div>
         </span>
+
         <a class="text-decoration-none text-dark" href="#">
           <div class="d-flex justify-content-center align-items-center mx-2">
             <i id="friends" class="fa-solid fa-users"></i>
           </div>
         </a>
       </div>
+    </div>
```

Comment:
Can you delete this line (line 83)? Currently the Create Trip button is stretching across the page and it's because of this line.

<img width="698" alt="image" src="https://user-images.githubusercontent.com/114224311/204173217-66436c9d-d93c-47a2-acee-ddb83415add5.png">